data/projects/editor_support.md

```diff
@@ -13,6 +13,28 @@ plugins. For some editors like sublime and gedit, running tests is not easy and
 there is no api to do it. In such cases we  would want to use dogtail
 (a GUI tester) or a similar tool to do this.
 
+Other details (for testing): 
+
+For sublime, we'll be using the unittest plugin which is a plugin by itself. 
```

Comment:
@Udayan12167 would the link be to the github repo of the framework or the official site of the framework?